data/book_server.json

```diff
@@ -0,0 +1,2184 @@
+{
+    "source_analysis": {
+        "source": "Source code",
+        "target": [],
+        "appName": "java-example-app"
+    },
+    "source_analysis_on_bookserverapp": {
```

Comment:
since we have these keys to get data for a particular application , we can name to have a generic name like ci_analysis.json